CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+# Contributing
+
+Please follow these steps:
+
+1. Fork this repository on GitHub
+1. Clone your fork
+1. Ensure all tests pass before making changes (`mvn test`)
+1. Make your changes on a topic branch off of master
+    - Write tests for your changes, and make sure *all* tests pass
+    - Commits should be atomic
+    - Commit messages should be in the following format:
+    
+            Short summary of the change
+        
+            Longer, more detailed message if needed. This may span multiple
+            paragraphs but each line should be no longer than 80 characters. 
```

Comment:
I think the 80 char limit is unnecessary with modern wrapped wording and githubs rendering of comments as markdown.